frontend/src/metabase-lib/metadata.ts

```diff
@@ -143,6 +144,23 @@ declare function DisplayInfoFn(
   stageIndex: number,
   segment: SegmentMetadata,
 ): SegmentDisplayInfo;
+/**
+ * Even though it seems weird to pass the same query two times,
+ * this function follows the same pattern as the other displayInfo functions.
+ * The first two parameters are always a query, and a stage.
+ * The third parameter is what you would like to have info about.
+ * It just only happens that the thing we're examining is (again) the query itself.
+ *
+ * @example
+ * const query = question.query();
+ * const { isEditable, isNative } = Lib.displayInfo(query, -1, query);
+ *
+ */
+declare function DisplayInfoFn(
+  _query: Query,
+  _stageIndex: number,
+  queryInfo: Query,
```

Comment:
just query
```suggestion
  query: Query,
```

Reply:
🤦

Reply:
Sorry about that.
Fixed it and removed JSDoc `@comment` in https://github.com/metabase/metabase/pull/37786/commits/265fb7d4826063158af1b680eafd4657ab546fb8